internal/rbd/rbd_util.go

```diff
@@ -1044,7 +1044,7 @@ func (rv *rbdVolume) hasSnapshotFeature() bool {
 }
 
 func (rv *rbdVolume) createSnapshot(ctx context.Context, pOpts *rbdSnapshot) error {
-	util.DebugLog(ctx, "rbd: snap create %s using mon %s", pOpts, pOpts.Monitors)
+	util.DebugLog(ctx, "rbd: snap create %s@%s using mon %s", rv, pOpts.String(), pOpts.Monitors)
```

Comment:
`%s@%s` and passing `rv, pOpts.String()` **should** be the same as just `%s` and `pOpts`. If it is not, that would be a bug (or very large inconsistency) in the caller of `createSnapshot`.

Or, if this is really how a rbdSnapshot does not match with its actual rbd-pool representation, introduce `rbdSnapshot.getSpec()` that returns the `<parentPool>/<parentName>@snapname` format.